lib/spack/spack/cmd/mark.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from __future__ import print_function
+
+import spack.cmd
+import spack.error
+import spack.package
+import spack.cmd.common.arguments as arguments
+import spack.installer
+import spack.repo
+import spack.store
+from spack.database import InstallStatuses
+
+from llnl.util import tty
+
+description = "mark packages as explicitly or implicitly installed"
+section = "admin"
+level = "long"
+
+error_message = """You can either:
+    a) use a more specific spec, or
+    b) use `spack uninstall --all` to uninstall ALL matching specs.
+"""
```

Comment:
The error message still refers to `spack uninstall`

Reply:
Thanks! I have pushed a new version where this is fixed.